learning-objectives/data.yml

```
@@ -131,6 +131,7 @@ firebase:
 front-end-web-dev:
   - components
   - state
+  - separation of concerns
```

```suggestion
  - separation-of-concerns
```

Ese pequeño cambio nomas habría que hacerle, pero además habría que agregar a [los archivos de internacionalización de OAs](https://github.com/Laboratoria/bootcamp/tree/main/learning-objectives/intl) un título y (ojalá) un link que explique el concepto, tanto en ES como PT!

Con eso estamos :smirk: 

¡Mil gracias @icarito y @Freihart512 ! :partying_face: